asg/configuration.tfvars

```diff
@@ -10,7 +10,7 @@ desired_capacity_on_demand = 2
 ec2_key_name = "key-name"
 instance_type = "t2.micro"
 minimum_healthy_percent_webapp = 50
-
+launch_type = "EC2" # can also be "FARGATE"
```

Comment:
`desired_capacity_on_demand`, `ec2_key_name`, and `instance_type` are also not needed for Fargate.

And also, I guess it makes more sense to use `us-east-1` in the `aws_region` example here (instead of `ap-northeast-1`).